types/webpack/index.d.ts

```diff
@@ -1832,7 +1833,45 @@ declare namespace webpack {
     }
 
     class ProgressPlugin extends Plugin {
-        constructor(options?: (percentage: number, msg: string, moduleProgress?: string, activeModules?: string, moduleName?: string) => void);
+        constructor(options?: ProgressPlugin.Options | ProgressPlugin.Handler);
+    }
+
+    namespace ProgressPlugin {
+        /**
+         * A handler function which will be called when webpack hooks report progress
+         */
+        type Handler = (percentage: number, msg: string, moduleProgress?: string, activeModules?: string, moduleName?: string) => void;
```

Comment:
Not sure about the `Handler` type here, the params `percentage` and `msg` are correct but could not find any documentation about the other parameters here.
The [documentation](https://github.com/webpack/webpack/blob/71eb5931dbd7db125109a95692065333cea936a9/declarations/plugins/ProgressPlugin.d.ts#L14) and the code uses
```ts
type Handler = (percentage: number, msg: string, ...args: string[]) => void;
```
So could you provide a link to the documentation where the other params are defined?
Think the string type is correct here, but your type would limit the number of arguments to 5 while I found no evidence that the number of args could not be greater 5.

Reply:
@ofhouse
Thank you so much. I feel apologized for the mistakes.

1. Default values were wrongly documented, because I only referred to official document
(https://webpack.js.org/plugins/progress-plugin/), which is different on v4 and v5 options. It is my fault. 😂

2. Actually I am also confused why the handler is of this signature. I just copied the function interface from previous commit, as I cannot find any reason to change an existing interface.

As you are suggesting here, therefore, I changed the function interface to your version, I think it is okay because (1) it conforms to the official document and the source code (2) the **...args: string[]** is a more loose version of three specific strings, which is compatible to previous usage, no breaking.